src/views/NotificationButton.tsx

```diff
@@ -124,12 +124,28 @@ class NotificationButton extends ComponentEx<IProps, IComponentState> {
       });
     }
 
+    const isRunningBgProcess = 
```

Comment:
Please don't adopt nonsense designer language into the code. A process is a well established term in programming and this has nothing to do with processes.